TESTING.md

```diff
@@ -48,9 +48,6 @@ Bigtable integration tests can either be run against an emulator or a real Bigta
 target environment can be selected via the `bigtable.env` system property. By default it is set to
 `emulator` and the other option is `prod`.
 
-To use the `emulator` environment, please install the gcloud sdk and use it to install the
```

Comment:
Should we add these instructions to the emulator package?

Reply:
No, these instructions are for running the tests. The emulator package has instructions focusing on how a customer can use the emulator for their own tests:

https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-testing/google-cloud-bigtable-emulator